theme/src/resources/templates/category_pages/liferay_dxp_7_2.hbs

```diff
@@ -0,0 +1,265 @@
+<div class="container-fluid container-fluid-max-xl main-content">
+	<div class="row">
+		<div class="col-12">
+			{{breadcrumbs}}
+		</div>
+	</div>
+
+	<div class="categories-main row">
+		<div class="category-heading col-md-12">
+			<h1>
+				{{dc '7_1-page_heading'}}
+			</h1>
+
+			{{#if category.description}}
+				<p class="secondary-font secondary-text-color">
+					{{category.description}}
+				</p>
```

Comment:
Can we change this to a `<div />` instead of a `<p />`?

Reply:
8883e18